spyderlib/qt/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 import os
 
-os.environ.setdefault('QT_API', 'pyqt')
+os.environ.setdefault('QT_API', 'pyqt5')
```

Comment:
how is it experimental? what is needed to get rid of that status?

and if pyqt5 isn’t installed, it will fall back to pyqt4, right?